### ISSUE TITLE
Typo in documentation about OTHER data type

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#370]
```
There is a little typo in the documentation here:
http://www.h2database.com/html/datatypes.html#other_type

IS:
Deserialization is only done get getObject is called. 

SHOULD BE:
Deserialization is only done when getObject is called. 
```

Original issue reported on code.google.com by `lukas.eder@gmail.com` on 2 Jan 2012 at 7:08

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the trunk now.
```

Original comment by `thomas.t...@gmail.com` on 6 Jan 2012 at 11:16
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in version 1.3.164
```

Original comment by `thomas.t...@gmail.com` on 3 Feb 2012 at 2:15
- Changed state: **Fixed**
